app/models/katello/concerns/redhat_extensions.rb

```diff
@@ -55,7 +51,7 @@ def kickstart_repos(host)
 
       def variant_repo(host, variant)
         if variant && host.content_source
-          product_id = host&.content_facet&.kickstart_repository&.product_id
+          product_id = host.try(:content_facet).try(:kickstart_repository).try(:product_id) || host.try(:kickstart_repository).try(:product_id)
```

Comment:
safe navigation operator fails when the object doesn't respond to the message!

Reply:
:+1: